### ISSUE TITLE
Filters on GET operation not added to docs

### ISSUE THREAD

**Issue by dannyvw** [api-platform/core#5527]
**Description**  
Filters applied on GET operation are not added to the docs. It is only applied for CollectionOperationInterface https://github.com/api-platform/core/blob/main/src/OpenApi/Factory/OpenApiFactory.php#L271

Is there any reason that this is only applied for CollectionOperationInterface?

**Comment by soyuka** [comment]
Indeed, in 2.6 this was not supported but now it should work as well with simple get operations. I'm not sure I want to fix this quite yet as I want the filters to be refactored for next year. Basically https://github.com/api-platform/api-platform/discussions/1724, probably also handle some kind of validation (we have this but architecture is quite distant from filtering) and transformation (https://github.com/symfony/symfony/pull/49134) from the query parameters may be of some interest. 

I'll be alright with a fix on the documentation but it'll not work with doctrine (and will not be supported by the default providers for now).

**Comment by atomiix** [comment]
I think the issue is not solved, or maybe I'm missing something but in my case, using api plarform v3.2.16 I still don't have the filter in the doc.

Here's my resource:

```php
<?php

declare(strict_types=1);

namespace App\Catalog\Infrastructure\Symfony\Resource;

use ApiPlatform\Metadata\ApiFilter;
use ApiPlatform\Metadata\ApiProperty;
use ApiPlatform\Metadata\Get;
use ApiPlatform\Metadata\GetCollection;
use ApiPlatform\Serializer\Filter\PropertyFilter;

#[Get]
#[GetCollection]
#[ApiFilter(PropertyFilter::class)]
class Menu
{
    #[ApiProperty(identifier: true)]
    public ?string $uuid;
    public string $name;
    public array $tags = [];
    public array $channels = [];
}
```

Here's the generated doc:
<img width="1349" alt="Capture d’écran 2024-03-13 à 15 49 49" src="https://github.com/api-platform/core/assets/2168836/741d637e-398d-41a6-bdc4-8113d9c4f99a">
<img width="1343" alt="Capture d’écran 2024-03-13 à 15 50 01" src="https://github.com/api-platform/core/assets/2168836/aac5a121-a721-40bf-b491-08ddb9a4dac2">

**Comment by soyuka** [comment]
fixed with parameters.
